package.json

```diff
@@ -1,32 +1,45 @@
 {
   "name": "serverless-log-forwarding",
   "version": "2.0.0",
+  "engines": {
+    "node": ">=12"
+  },
   "description": "a serverless plugin to forward logs to given lambda function",
-  "main": "index.js",
+  "main": "dist/src/index.js",
   "directories": {
     "test": "test"
   },
   "dependencies": {
-    "underscore": "^1.8.3"
+    "@types/underscore": "^1.11.4",
```

Comment:
For what is needed `@types/underscore`?

Reply:
As I understand the prefix `@types` it's something related to the typescript code validation/highlighting, right?

Reply:
addressed